src/EditorFeatures/Core/Implementation/Suggestions/SuppressionSuggestedAction.cs

```diff
@@ -33,8 +34,8 @@ internal class SuppressionSuggestedAction : SuggestedAction, ITelemetryDiagnosti
             IWaitIndicator waitIndicator,
             CodeFix fix,
             object provider,
-            Func<CodeAction, SuggestedActionSet> getFixAllSuggestedActionSet) :
-                base(workspace, subjectBuffer, editHandler, waitIndicator, fix.Action, provider)
+            Func<CodeAction, SuggestedActionSet> getFixAllSuggestedActionSet)
+            : base(workspace, subjectBuffer, editHandler, waitIndicator, fix.Action, provider)
```

Comment:
Well, if you care enough about this to move the colon to the next line, then indent it to match the more common style of this in the codebase.